src/test/rpc/AccountInfo_test.cpp

```diff
@@ -188,6 +188,127 @@ class AccountInfo_test : public beast::unit_test::suite
         }
     }
 
+    // Test the "signer_lists" argument in account_info, with api_version 2.
+    void
+    testSignerListsApiVersion2()
```

Comment:
Is it intended that `testSignerListsApiVersion2()` not be run?

Reply:
Thanks for pointing that out, did think it a bit odd "rippled -u" passed so easily. I've fixed up the test so it will pass if ApiMaximumSupportedVersion is set to 2 and I've added the call to the test commented out with an explanation. Will wait to hear more from @scottschurr  and @pwang200 on how to actually write tests for the v2 api.